tests/compiletest.rs

```diff
@@ -106,6 +106,8 @@ regexes! {
     "[^ `]*/(rust[^/]*|checkout)/library/" => "RUSTLIB/",
     // erase platform file paths
     "sys/[a-z]+/"                    => "sys/PLATFORM/",
+    // erase error annotations
+    "//(\\[[a-z,]+\\])?~.*"                    => "",
```

Comment:
We had deliberately removed that from here since ui_test should take care of its own annotations itself?

Reply:
that's because most of them disappeared with the jsonification, not because we explicitly erase them.

Reply:
EDIT: unless it's `-Zui-test` that does something here inside rustc, checking

Reply:
jsonification has nothing to do with the stderr files...?

Reply:
> unless it's -Zui-test that does something here inside rustc, checking

We still filter them out here:

https://github.com/rust-lang/miri/blob/248a10eaf5cc6df2491a6e708524a41503674ffb/ui_test/src/rustc_stderr.rs#L118-L121

I am not quite sure what else you are looking for.^^

Reply:
I was grepping for `//~` 🤦